### PR TITLE
Minor fix in `postLoad` event (file uploading)

### PR DESCRIPTION
**Pull request by icamys**
Minor fix in `postLoad` event (file uploading). Added accessor methods in FileUploader class to make it possible to get $targetDir from outside.

### CODE REVIEW

**Review by roman-1983 — APPROVED** [review]
Changes look good.

File: controller/upload_file.rst [hunk 1]
```diff
@@ -243,6 +243,16 @@ logic to a separate service::
 
             return $fileName;
         }
+
+        public function getTargetDir()
+        {
+            return $this->targetDir;
+        }
+
+        public function setTargetDir($targetDir = '')
```
Comment (Simperfit) [inline controller/upload_file.rst]: default empty string ?

Comment (icamys) [inline controller/upload_file.rst]: Fair observation. Perhaps it would be better to move default empty string to property declaration?

Comment (xabbuh) [inline controller/upload_file.rst]: I would just remove the setter. IMO there's no need to have it. We only need the getter to be able to read the configured directory later.

**Review by xabbuh — APPROVED** [review]

File: controller/upload_file.rst (hunk 1 above)
Comment (icamys) [inline controller/upload_file.rst]: @xabbuh Done.

**Review by javiereguiluz — APPROVED** [review]
👍

**Comment by xabbuh** [comment]
Thank you @icamys.

**Pull request closed without merging**